lib/laa_crime_schemas/types/types.rb

```diff
@@ -298,6 +298,14 @@ module Types
     InterestsOfJusticeResult = String.enum(TestResult['passed'], TestResult['failed'])
     FundingDecision = String.enum(*%w[granted refused])
 
+    AssessmentRules = String.enum(*%w[
```

Comment:
Do you think there might be confusion with the assessment rules term and the changes we're going to make with the means test review